src/transaction/types.ts

```diff
@@ -6,12 +6,13 @@ import type {
   Hex,
   TransactionReceipt,
 } from 'viem';
+import type { WalletCapabilities as ViemWalletCapabilities } from 'viem';
 import type { Config } from 'wagmi';
 import type {
   SendTransactionMutateAsync,
   WriteContractMutateAsync,
 } from 'wagmi/query';
-import type { WalletCapabilities as OnchainKitWalletCapabilities } from '../types';
+// 🌲☀🌲
```

Comment:
Not big deal, but this is an easter egg that should alway be as first line only :)

So you probably want to remove it from here